frontend/src/metabase/modes/components/drill/ObjectDetailDrill.unit.spec.ts

```diff
@@ -10,27 +9,66 @@ import {
   SAMPLE_DB_ID,
 } from "metabase-types/api/mocks/presets";
 import { TYPE as SEMANTIC_TYPE } from "cljs/metabase.types";
+import { checkNotNull } from "metabase/core/utils/types";
+import type {
+  QuestionChangeClickAction,
+  ReduxClickAction,
+  ClickObject,
+} from "metabase/modes/types";
+import type {
+  StructuredDatasetQuery,
+  DatasetColumn,
+  RowValue,
+} from "metabase-types/api";
+import type { State } from "metabase-types/store";
+
 import Question from "metabase-lib/Question";
+import type StructuredQuery from "metabase-lib/queries/StructuredQuery";
+import { ObjectDetailDrill } from "./ObjectDetailDrill";
 
 const metadata = createMockMetadata({
   databases: [createSampleDatabase()],
 });
 
-const ordersTable = metadata.table(ORDERS_ID);
-const orderIDField = metadata.field(ORDERS.ID);
-const orderIDColumn = metadata.field(ORDERS.ID).column();
+const ordersTable = checkNotNull(metadata.table(ORDERS_ID));
+const orderIDField = checkNotNull(metadata.field(ORDERS.ID));
+const orderIDColumn = orderIDField.column();
 
 const DEFAULT_CELL_VALUE = 1;
 
+const DATA_VALUES = {
+  [ORDERS.ID]: "5",
+  [ORDERS.USER_ID]: "1",
+  [ORDERS.PRODUCT_ID]: "132",
+  [ORDERS.SUBTOTAL]: 127.88197029833711,
+  [ORDERS.TAX]: 7.03,
+  [ORDERS.TOTAL]: 134.9119702983371,
+  [ORDERS.DISCOUNT]: null,
+  [ORDERS.CREATED_AT]: "2018-10-10T03:34:47.309+03:00",
+  [ORDERS.QUANTITY]: 5,
+};
+
+const DATA = ordersTable.fields?.map(field => ({
+  col: field.column(),
+  value: DATA_VALUES[field.column().id as number],
```

Comment:
Another solution would be to type `DATA_VALUES` like `Record<string, unknown>` or something like that